src/lib/select/select.html

```diff
@@ -5,11 +5,12 @@
   <span class="md-select-arrow"></span>
 </div>
 
-<template connected-overlay [origin]="origin"  [open]="panelOpen" hasBackdrop (backdropClick)="close()"
+<template connected-overlay [origin]="origin" [open]="panelOpen" hasBackdrop (backdropClick)="close()"
   backdropClass="md-overlay-transparent-backdrop" [positions]="_positions" [minWidth]="_triggerWidth"
   [offsetY]="_offsetY" [offsetX]="_offsetX" (attach)="_setScrollTop()">
   <div class="md-select-panel" [@transformPanel]="'showing'" (@transformPanel.done)="_onPanelDone()"
-    (keydown)="_keyManager.onKeydown($event)" [style.transformOrigin]="_transformOrigin">
+    (keydown)="_keyManager.onKeydown($event)" [style.transformOrigin]="_transformOrigin"
+      [ngClass]="_panelDoneAnimating ? 'md-select-panel-done-animating' : ''">
```

Comment:
Have you tried using a regular class binding?  It might be less verbose.

```html
[class.ng-select-panel-done-animating]="_panelDoneAnimating"
```